.readthedocs.yaml

```diff
@@ -11,9 +11,13 @@ sphinx:
   configuration: docs/conf.py
   fail_on_warning: true
 
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
```

Comment:
The RTD configuration should be redone to use mamba, see https://github.com/astropy/astropy/blob/main/.readthedocs.yaml for an example